test/controllers/GovernmentSpendControllerTest.scala

```diff
@@ -102,185 +110,182 @@ class GovernmentSpendControllerTest extends UnitSpec with FakeTaxsPlayApplicatio
+      document.select("#national_debt_interest").text() should include("7.0%")
       document.select("#defence + td").text() shouldBe "£1,269.73"
-      document.select("#defence").text() should include ("5.31%")
+      document.select("#defence").text() should include("5.31%")
       document.select("#criminal_justice + td").text() shouldBe "£1,052.13"
-      document.select("#criminal_justice").text() should include ("4.4%")
+      document.select("#criminal_justice").text() should include("4.4%")
       document.select("#transport + td").text() shouldBe "£705.40"
-      document.select("#transport").text() should include ("2.95%")
+      document.select("#transport").text() should include("2.95%")
       document.select("#business_and_industry + td").text() shouldBe "£655.19"
-      document.select("#business_and_industry").text() should include ("2.74%")
+      document.select("#business_and_industry").text() should include("2.74%")
       document.select("#government_administration + td").text() shouldBe "£490.20"
-      document.select("#government_administration").text() should include ("2.05%")
+      document.select("#government_administration").text() should include("2.05%")
       document.select("#Culture + td").text() shouldBe "£404.11"
-      document.select("#Culture").text() should include ("1.69%")
+      document.select("#Culture").text() should include("1.69%")
       document.select("#Environment + td").text() shouldBe "£396.94"
-      document.select("#Environment").text() should include ("1.66%")
+      document.select("#Environment").text() should include("1.66%")
       document.select("#HousingAndUtilities + td").text() shouldBe "£392.16"
-      document.select("#HousingAndUtilities").text() should include ("1.64%")
+      document.select("#HousingAndUtilities").text() should include("1.64%")
       document.select("#overseas_aid + td").text() shouldBe "£274.99"
-      document.select("#overseas_aid").text() should include ("1.15%")
+      document.select("#overseas_aid").text() should include("1.15%")
       document.select("#uk_contribution_to_eu_budget + td").text() shouldBe "£179.34"
-      document.select("#uk_contribution_to_eu_budget").text() should include ("0.75%")
+      document.select("#uk_contribution_to_eu_budget").text() should include("0.75%")
 
       document.getElementById("user-info").text() should include("userForename userSurname")
-      document.getElementById("user-info").text() should include("Unique Taxpayer Reference: "+testUtr)
+      document.getElementById("user-info").text() should include("Unique Taxpayer Reference: " + testUtr)
       document.select("#gov-spend-total + td").text() shouldBe "£23,912.00"
       document.select(".page-header h1").text shouldBe "Tax year: April 6 2013 to April 5 2014 Your taxes and public spending"
     }
 
     "have correct data for 2015" in new TestController {
 
-      override val model = new GovernmentSpend(
+       val model2 = new GovernmentSpend(
```

Comment:
Should this not still be an override of val model? The trait is constructed so that on instance it will add a mock to return whatever is assigned to the model value